src/lib/rpc.ts

```diff
@@ -108,6 +120,131 @@ export const makeRpcUtils = async ({ agoricNet }) => {
     leader,
     vstorage,
     storageWatcher,
+    agoricNet,
+    netConfigURL,
+    networkConfig,
   };
 };
 export type RpcUtils = Awaited<ReturnType<typeof makeRpcUtils>>;
+
+export enum LoadStatus {
+  Idle = 'idle',
+  Waiting = 'waiting',
+  Received = 'received',
+}
+
+export const usePublishedKeys = (path: string) => {
```

Comment:
please add a jsdoc of what this is for. just from the name, "keys" sounds like crypto key material. also "published" doesn't convey the vstorage path. 

why even make this function concerned with `published.`"? That's not much burden on the caller.

```suggestion
export const useVstorageChildKeys = (path: string) => {
```

Reply:
Done